app/src/main/java/org/wikipedia/analytics/ABTest.kt

```diff
@@ -20,7 +20,7 @@ open class ABTest(private val abTestName: String, private val abTestGroupCount:
 
     protected open fun assignGroup() {
         testGroup = Random(System.currentTimeMillis()).nextInt(Int.MAX_VALUE).mod(abTestGroupCount)
-        ActivityTabEvent.submit(activeInterface = "NULL/NOT NEEDED", action = "group_assign", group = when (testGroup) {
+        ActivityTabEvent.submit(activeInterface = "null", action = "group_assign", group = when (testGroup) {
```

Comment:
I see the `activeInterface` is a non-null type of variable. Could you please confirm with Shay that we should send a "null" string or that an empty string (`""`) is fine?